legion/tests/e2e/robot/tests/1_common/1.0_check_components_auth.robot

```diff
@@ -10,22 +10,24 @@ Test Setup          Choose cluster context            ${CLUSTER_CONTEXT}
 
 *** Keywords ***
 
-Url stay the same after dex log in
```

Comment:
"    Should contain   ${resp.text}    Invalid Email Address and password"
Must be updated to Keycloak pattern